src/styles/header.scss

```diff
@@ -0,0 +1,79 @@
+:root {
+  --main-color: #00ACDC;
+}
```

Comment:
You have already created a variable with this value in another file, so it would be better to export the variables to a separate file and use that file everywhere